src/main/java/com/google/devtools/build/lib/analysis/actions/PathMappers.java

```diff
@@ -64,16 +68,21 @@ public final class PathMappers {
    * sets and thus can't result in memory regressions.
    *
    * @param mnemonic the mnemonic of the action
+   * @param executionInfo the execution info of the action
    * @param outputPathsMode the value of {@link CoreOptions#outputPathsMode}
    * @param fingerprint the fingerprint to add to
    */
   public static void addToFingerprint(
-      String mnemonic, OutputPathsMode outputPathsMode, Fingerprint fingerprint) {
+      String mnemonic,
+      Map<String, String> executionInfo,
+      OutputPathsMode outputPathsMode,
+      Fingerprint fingerprint)
+      throws CommandLineExpansionException, InterruptedException {
```

Comment:
does this method actually throw any exception?

Reply:
It doesn't. For some reason my IDE thought it did.